datajs/src/parser/csv.js

```diff
@@ -31,7 +31,6 @@ export async function guessParseOptions(file) {
   } else if (file.displayName === 'FileInterface') {
     let reader = file.descriptor.stream().getReader()
     text = await toNodeStream(reader, 10, true)
```

Comment:
I guess this is the only place where we use `toNodeStream()`, can we try regular `stream()` function? Like we do in the above if case? check line 28

Reply:
Once this is done, I think we can completely remove `toNodeStream()`

Reply:
Using a regular stream function to achieve the function of toNodeStream() is quite tricky. I spent a lot of time trying to make this work, but most solutions tend towards the existing implementation.

Reply:
So this is related to above comment right?

Reply:
Yeah, I fixed it!